.travis.yml

```diff
@@ -2,11 +2,21 @@ language: ruby
 cache: bundler
 before_install:
   - gem install bundler
-script:
-  - bundle exec jekyll algolia --config=_config.yml,algolia_config/_marketplace_dev.yml
-  - bundle exec jekyll algolia --config=_config.yml,algolia_config/_marketplace_user.yml
-branches:
-  only:
-    - master
+
 rvm:
   - 2.5
+
+jobs:
+  include:
+    - name: Page validation
+      script:
+        - bundle exec jekyll validate
+      before_install:
+        - gem install bundler
+
+    - name: Push pages to Algolia
```

Comment:
This build must work with master branch only

Reply:
Adressed.